bot/exts/help_channels/_cog.py

```diff
@@ -410,15 +418,20 @@ async def move_to_in_use(self, channel: discord.TextChannel) -> None:
             category_id=constants.Categories.help_in_use,
         )
 
-        timeout = constants.HelpChannels.idle_minutes * 60
+        timeout = constants.HelpChannels.idle_minutes_others * 60
```

Comment:
Shouldn't it use `idle_minutes_claimant` since the message responsible for this was sent by the claimant?

Reply:
yep, definitely should be claimant.
```suggestion
        timeout = constants.HelpChannels.idle_minutes_claimant * 60
```